### DISCUSSION TITLE
Verification and Validation of my model

### DISCUSSION THREAD

**Discussion by LordMicidial** [firemodels/fds#12718] [Q&A]
Dear all, 

I've basically completed the geometry of a model which consists of an almost rectangular-shaped warehouse which was used for logistics. 

In the future, the property would like to host other activities (storage of industrial kitchens and/or polyethene film). Hence, all I have to do now is to design cubic-shaped pallet-like items which represent the storage goods and set realistic fires which might happen.

I will then model the sprinkler activation by cutting the HRR curve as it is legit according to my country's legislation to simplify the modelling.

The aim of the study is life-safety, I just need to represent people escaping the fire easily.

Concerning model Validation, is it legit to just extrapolate the Material Properties of the item I must model from Material Safety Datasheet (e.g. polyethilene film)? There are no case that fit mine precisely in the Guide and off course I can't project an experiment just to prove my model. Otherwise can you refer to other sources or database?

Concerning Verification I have performed grid sensitivity looking for Temperature with Mesh of 0.4, 0.2, 0.1. With 0.2 and 0.1 results are very similar in the points where I am measuring the T. Can I just stop and assume the model will behave correctly even when I will have find the MATL property to describe the Materials from Safety Datasheets?

Regards

**Comment by drjfloyd** [comment]
If you are prescribing the fire, then your grid study is probably fine. If you have prescribed the fire in your grid study but intend to predict the fire in the actual simulation, then you should have some kind of grid study for the prediced heat release.

There are many sources for material properties: SFPE Handbook, materials.fsri.org, janaf.nist.gov, webbook.nist.gov, matweb.com, using your favorite search engine, etc.

Modeling questions aside, however, you mave have a problem with the assumption that sprinklers will control the fire. Warehouse sprinklering is designed for the commodity classification of what is being stored in the warehouse. If the current sprinkler design is for a lower commodify classification than the new use, it may not provide sufficient water to control the fire.

**Comment by LordMicidial** [comment]
Thank you for the quick response!

Yes. I will enter a value for the HRRPUA of the ignition source and the MATL_ID associated with the elements that will catch fire due to reaching the ignition temperature. So as per the Verification Guide the fires are prescribed, **so the grid study is considered sufficient.** 

Thank you also for the sources I was checking the SFPE only.

Your last concern about the sprinkler system is more than legitimate, however, given that models will run to determine the safety of a detailed scenario I can't control the property to comply with the safety managent system we are designing for them. Probably it is best to exaggerate the sizing of the sprinkler system but this is not a modelling task.

So given the relatively easiness of the model wouldn't you suggest to double check it against a particular validation scenario?

I'll then close this Q&A thread.

regards

**Reply by drjfloyd** [reply]
You stated that your modeling will assume that sprinklers control the fire. That assumption relies on the sprinkler design being adequate for the commodity class being stored. If the warehouse has an existing sprinkler system that is not being replaced, you need to make sure that its design is adeqaute for the new usage.

**Reply by LordMicidial** [reply]
There are no sprinkler right now, however I will report this issue to the Project Manager so he can grasp the whole picture. Thank you!
